### PR TITLE
delete obsolete calc_damage endpoint and also pull out routing logic

### LINKED ISSUE #53: stop lying about gcd and mp consumption

### PR DESCRIPTION
**Pull request by ackao**
fixes #53

also begins to address breakdown of routing/backend app logic #39

### CODE REVIEW

**Review by redshadowhero — CHANGES_REQUESTED** [review]
a nit and a conversation item.

File: backend/xivdpscalc/routing.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
+from flask import Flask, jsonify, request, abort
+from flask_cors import CORS
```
[the remark this answers is not in the document]
Comment (ackao) [reply backend/xivdpscalc/routing.py:2]: durr. will fix.

File: backend/xivdpscalc/routing.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+            'job': string
+            'comp': Array}
+    output: {'dps': float,
+             'gcd': float,
+             'mp': float}
+    """
+    data = request.get_json()
+
+    if not data:
+        return "abort", abort(400)
```
[the remark this answers is not in the document]
Comment (ackao) [reply backend/xivdpscalc/routing.py:32]: that's a new github issue :D

[the remark this answers is not in the document]
Comment (ackao) [reply backend/xivdpscalc/routing.py:32]: honestly all error handling needs a fix

[the remark this answers is not in the document]
Comment (redshadowhero) [reply backend/xivdpscalc/routing.py:32]: alright, I'll close this then. I figured it was worth discussion anyway.

**Review by redshadowhero — APPROVED** [review]
🚢 ➡️ 📦

**Pull request merged**